Makefile

```diff
@@ -30,6 +30,14 @@ test-e2e:
 	./openshift/e2e-tests.sh
 .PHONY: test-e2e
 
+perf-tests:
+	export ES_DEVELOPMENT=true && \
+    export ES_HOST_PORT=search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com && \
+    export USE_OPEN_SEARCH=true && \
```

Comment:
Do we need these to be exported? Probably
`ENABLE_INTERNAL_TLS="true" ./openshift/e2e-tests.sh`
like below would also work?

Reply:
I will check.